sim/core/armor_test.go

```diff
@@ -224,39 +224,39 @@ func TestDamageReductionFromArmor(t *testing.T) {
 	sporeCloudAura.Deactivate(&sim)
 	shatteringThrowAura.Deactivate(&sim)
 	attackTable.UpdateArmorDamageReduction()
-	expectedDamageReduction = 0.3468
+	expectedDamageReduction = 0.6532
 	if !WithinToleranceFloat64(expectedDamageReduction, attackTable.ArmorDamageReduction, tolerance) {
 		t.Fatalf("Expected major & minor armor modifier to result in %f damage reduction got %f", expectedDamageReduction, attackTable.ArmorDamageReduction)
 	}
 
 	// Cap armor pen
 	attacker.stats[stats.ArmorPenetration] = 1400
 	attackTable.UpdateArmorDamageReduction()
-	expectedDamageReduction = 0.0203
+	expectedDamageReduction = 0.9797
```

Comment:
The new test results don't seem intuitive. An attacker with 1400 armor pen and all the debuffs should be around 2% damage reduction from armor. The values we are getting from the test now are just 1 - the original value. This makes sense based on the change to the implementation but doesn't follow the damage reduction semantics it should.